team13-fydp/fydp
# Output Teacher Overview + Appearance

Alyssa created the output teacher overview and I made some changes to it so that it is readable.

Some minor fixes were also needed to fix errors (i.e. just use cohortNames.size())

To test:
- run model
- open flamborough sheet, navigate to last output (just created)
- check to make sure text is wrapped, top row with day names is bolded and all model results expected (i.e. cohort name and subject appear in matrix), teacher names as column
- make sure that 'prep' and 'away' are listed in each applicable cell, without any slashes or other information
